livekit-rtc/livekit/rtc/chat.py

```diff
@@ -0,0 +1,140 @@
+# Copyright 2023 LiveKit, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from dataclasses import dataclass, field
+from datetime import datetime
+import json
+import logging
+from typing import Any, Callable, Dict, Optional
+
+from .room import Room, Participant, DataPacket
+from ._proto.room_pb2 import DataPacketKind
+from ._utils import generate_random_base62
+
+_CHAT_TOPIC = "lk-chat-topic"
+_CHAT_UPDATE_TOPIC = "lk-chat-update-topic"
+
+
+class ChatManager:
```

Comment:
Should this class also accumulate chat messages with a sane default way of updating chat messages (i.e. setting deleted from false to true)?

Reply:
yeah I def want to in the future.. just wanted to keep it simple because if we started buffering, we'd need a way to clean it up too to avoid it buffering forever